config/schema/api.json

```diff
@@ -0,0 +1,1012 @@
+          "format": "date-time",
+          "type": [
+            "string"
+          ],
+          "example": "2013-09-12 22:49:27 +0530"
+        },
+        "identity": {
+          "anyOf": [
+            {
+              "$ref": "#/definitions/game_night/definitions/id"
+            }
+          ]
+        },
+        "created_at": {
+          "description": "when game_night was created",
+          "format": "date-time",
+          "type": [
+            "string"
+          ]
+        },
+        "updated_at": {
+          "description": "when game_night was updated",
+          "format": "date-time",
+          "type": [
+            "string"
+          ]
+        }
+      },
+      "links": [
+        {
+          "description": "Create a new game_night.",
+          "href": "/game_nights",
+          "method": "POST",
+          "rel": "create",
+          "http_header": {
+            "Authorization": "Bearer: iOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE0NTM5MzkwMjYsImF1ZCI6bnVsbCwic3ViIjoxfQ.TSgJie1vAWxR_ym_uazjKs43rpQ50OKmb2KZ2N6ALs4"
+          },
+          "schema": {
+            "properties": {
+              "id": {
+                "$ref": "#/definitions/game_night/definitions/id"
+              },
+              "location_name": {
+                "$ref": "#/definitions/game_night/definitions/location_name"
+              },
+              "location_address": {
+                "$ref": "#/definitions/game_night/definitions/location_address"
+              },
+              "time": {
+                "$ref": "#/definitions/game_night/definitions/time"
+              }
+            },
+            "required": [
+              "id",
+              "time",
+              "location_name"
+            ],
+            "type": [
+              "object"
+            ]
+          },
+          "title": "Create",
+          "targetSchema": {
+            "type": [
+              "object"
+            ],
+            "properties": {
+              "id": {
+                "$ref": "#/definitions/game_night/definitions/id"
+              },
+              "location_name": {
+                "$ref": "#/definitions/game_night/definitions/location_name"
+              },
+              "location_address": {
+                "$ref": "#/definitions/game_night/definitions/location_address"
+              },
+              "time": {
+                "$ref": "#/definitions/game_night/definitions/time"
+              }
+            }
+          }
+        },
+        {
+          "description": "Delete an existing game_night.",
+          "href": "/game_nights/{(%23%2Fdefinitions%2Fgame_night%2Fdefinitions%2Fidentity)}",
+          "method": "DELETE",
+          "rel": "destroy",
+          "http_header": {
+            "Authorization": "Bearer: iOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE0NTM5MzkwMjYsImF1ZCI6bnVsbCwic3ViIjoxfQ.TSgJie1vAWxR_ym_uazjKs43rpQ50OKmb2KZ2N6ALs4"
+          },
+          "title": "Delete"
+        },
```

Comment:
@Enriikke so for deleting records, the response we're going for is 204/no-content. I'm not entirely sure how to represent this though. Currently it just looks for all of the properties on the appropriate model --

```json
{
  "created_at": "2015-01-01T12:00:00Z",
  "id": 1294,
  "location_name": "Silver Spring Library",
  "location_address": "8401 Fenton st. Silver Spring, MD 20910",
  "time": "2013-09-12 22:49:27 +0530",
  "updated_at": "2015-01-01T12:00:00Z"
}
```